Chat_App/Core/Custom UI/TextView/TextView.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  TextView.swift
+//  Chat_App
+//
+//  Created by Ana Dzamelashvili on 5/11/23.
+//
+
+import Foundation
+import UIKit
+
+protocol ButtonActionDelegate: AnyObject {
+    func buttonTapped(sender: UIButton)
+}
+protocol TextViewDelegate: AnyObject {
+    func changeTextColor(color: UIColor)
+}
+
+class TextView: UIView {
+    
+    private let lineHeight: CGFloat = Constants.TextView.font.lineHeight
+    private lazy var sendButton = UIButton()
+    private lazy var containerView = UIView()
+    lazy var textView = UITextView()
```

Comment:
make it public private(set), you probably don't need to set that textView from the outside